rainbow-me/rainbowkit
# feat: enhanced provider API / `@tanstack/react-query` peer dependency

## Changes
- Added `@tanstack/react-query` as a peer dependency and introduced enhanced provider API.
- Implemented fallback to fetch ENS name from enhanced provider API when mainnet is unavailable.

## What to test
- Go to this preview link: https://rainbowkit-example-h3v2divmt-rainbowdotme.vercel.app/
- Open up network tab by right clicking -> inspect element -> network tab
- If you connect to a wallet that has ENS name you should be able to see it making request to our enhanced provider API
